src/components/GenericWarningModal/GenericWarningModal.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import { Button, useDisclosure } from "@chakra-ui/react"
+import { Story, ComponentMeta } from "@storybook/react"
+
+import { GenericWarningModal } from "./GenericWarningModal"
+
+const modalMeta = {
+  title: "Components/GenericWarningModal",
+  component: GenericWarningModal,
+} as ComponentMeta<typeof GenericWarningModal>
+
+interface TemplateArgs {
+  buttonText: string
+  displayTitle: string
+  displayText: string
+}
+
+const SingleButtonModalTemplate: Story<TemplateArgs> = ({
+  buttonText,
+  displayTitle,
+  displayText,
+}: TemplateArgs) => {
+  const { isOpen, onOpen, onClose } = useDisclosure({ defaultIsOpen: true })
+  return (
+    <>
+      <Button onClick={onOpen}>Open Modal</Button>
+      <GenericWarningModal
+        isOpen={isOpen}
+        onClose={onClose}
+        displayTitle={displayTitle}
+        displayText={displayText}
+      >
+        <Button>{buttonText}</Button>
```

Comment:
we should aim to provide the most commonly used variant (`danger`)

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/941/commits/d669dc200631766d1204621f55ffd23099f4829f